pkg/utils/uuid.go

```diff
@@ -0,0 +1,50 @@
+package utils
+
+import (
+	"math/rand"
+	"time"
+)
+
+func GetUuid(l int) string {
+	initByte := []byte{}
+	result := []byte{}
+
+	for i := 48; i < 123; i++ {
+		switch {
+		case i < 58:
+			initByte = append(initByte, byte(i))
+		case i >= 65 && i < 91:
+			initByte = append(initByte, byte(i))
+		case i >= 97 && i < 123:
+			initByte = append(initByte, byte(i))
+		}
+
+	}
+
+	rand.Seed(time.Now().UnixNano())
```

Comment:
rand.Seed should be in init func, it only called once

Reply:
time.Now will change every call

Reply:
rand.Seed only need one random seed, the all next call `rand.XXX` are all random